e2e/rbd.go

```diff
@@ -417,6 +433,13 @@ var _ = Describe("RBD", func() {
 							if err != nil {
 								e2elog.Failf("failed to create PVC and application with error %v", err)
 							}
+							md5sumClone, err := calMd5sum(&a, f)
+							if err != nil {
+								e2elog.Failf("failed to calculate md5sum with error %v", err)
+							}
+							if md5sumClone == md5sum {
```

Comment:
Why this is a failure condition? If both are equal then it is correct, right?

Reply:
corrected.